projects/plugins/jetpack/images/jetpack-banner-gradient.svg

```diff
@@ -0,0 +1,33 @@
+<svg width="547" height="138" viewBox="0 0 547 138" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I'm not sure if this is the best approach for this particular use case. Some of the used SVG functions have unknown support in Safari (e.g. https://caniuse.com/?search=feFlood) and it looks a bit weird on mobile.
I'm guessing Crew will have a better idea of whether this is fine or if we should use a rasterized version of this.

Reply:
Great find! My own testing didn't show any issues and I wasn't aware of flood/gradient still being an issue in modern browsers, so thank you very much for this hint 🙇  
I've replace the svg with a png just to be on the safe side 🦺